public/locales/en/common.json

```diff
@@ -239,8 +239,8 @@
     "CENTER_CREATED_SUCCESSFULLY": "Center Created Successfully",
     "SEARCHBAR_PLACEHOLDER": "Search Center..",
     "CENTER_UPDATE_FAILED": "Something went wrong while updating center",
-    "ARCHIVED_MEMBERS": "Archived Members:",
-    "ACTIVE_MEMBERS": "Active Members"
+    "ARCHIVED_MEMBERS": "ARCHIVED MEMBERS",
```

Comment:
@AkshataKatwal16 here u can keep thing capitalize only... dont make it uppercase here... instead wherever required u do transform there to uppercase

Reply:
okay